backend/database/db_manager.py

```
@@ -1,45 +1,36 @@
 import os
-import sqlite3
+import sys
+import mariadb# importing necessary functions from dotenv library
+from dotenv import load_dotenv, dotenv_values
```

Ruff (0.8.2)

`dotenv.dotenv_values` imported but unused

Remove unused import: `dotenv.dotenv_values`

(F401)